### PR TITLE
Feature/general styles

### PR DESCRIPTION
**Pull request by fromITzerotoIThero**
I've changed some of the general styles, to make the app present better, here are some issues I've encountered (please ignore the ones you find unimportant as of now - after all, it kind of works, maybe I'll worry about it later):

1. Setting max-width: 360px on "body" causes it to move to the left - why? I would want to have it set, so as to prevent the app from any weird display issues. 

2. How to get rid of this "jumping" of the app when navigating to other pages? Is there any fast and easy way of going about it and what is the reason for it?

3. How to center the WelcomeScreen? - just now I thought I could try to do it with creating App.module.css back again and some flex - but I'm not sure how that would affect the PR (would it even show?).

### CODE REVIEW

**Comment by piotrwicijowski** [comment]
1. Setting it on the body means that making it a `flex` container (which was a method you used for centering) means that the container itself is narrow. If you want body to be 360px, then you can remove the `flex` container from it, and use a more old-school way of centering by using `margin: 0 auto` on the `<body>`
2. You get jumping, because you apply `360px` on elements such as `ProductPage`, and as a result the width is applied only once those elements get displayed (this happens after they load). If, instead, you applied it on something top level, like #root or <body>, then the problem would go away
3.  You can just adjust the margins of the BasicCard so that it gets centered and be done with it ;)

**Comment by fromITzerotoIThero** [comment]
So to sum up, you would advise me to:
- center the app with flex on body
- set max-width (or maybe width?) to 360px on #root 
- try to remove width on different child components (which I'm currently trying out, but feels more complicated than I thought 😝 )
? :)

Ad.2
Does it apply to 'width' with any type of value (360px, 100%, inherit) or only fixed pixels? (tried these options and the bouncing is still there, so I guess yes - am asking because I sense there some underlying issue with that)

**Comment by piotrwicijowski** [comment]
I may have misspoken - you don't need to remove width from any element, it was just that the jumping was caused by the width being applied only to those elements, and not e.g. to some top level element, like #root. Also, instead of using max-width, just use width, because with only max width you will run into cases where a less-wide content will make the whole thing shrink (and cause everything to jump).

**Comment by piotrwicijowski** [comment]
Can you push the changes where you apply 360 on #root?

**Comment by piotrwicijowski** [comment]
I don't see any more jumping after latest commit. Unless you are referring to some other type of jumping

**Comment by fromITzerotoIThero** [comment]
I still see it, when navigating to different views, it's better now (no change on the width of the app), but seems like the position is dancing slightly to the right - click on navigation bar or buttons to see :)

**Comment by piotrwicijowski** [comment]
Ah, I think I get it now - I was opening it in Firefox on Linux (which has a bit different behavior) - it is the scrollbar that disappears during loading (because there is not enough content for there be a need for scrollbar) and once content appears, the scrollbar also appears, requiring horizontal space, and shifting everything a bit to the left. Add a `scrollbar-gutter: stable;` to your `html` element in css and you should be good.

**Comment by fromITzerotoIThero** [comment]
W., I'm not sure what to do this conflict resolving thing... After that how can I merge it with master?

**Comment by piotrwicijowski** [comment]
Well, you need to resolve the conflicts. It seems a file was modified both in this branch, and at the same time for some reason in the master branch. I recommend that you familiarize yourself with conflict resolution, especially since the UI for it in github is quite convenient

**Comment by fromITzerotoIThero** [comment]
Thx! That's what I'm gonna do just now :)

**Comment by fromITzerotoIThero** [comment]
Argh!!! I f**ked it all up! 🤦 
With GitHub I made changes to the code, "Marked as resolved", "Commited merge" and the conflict was resolved, trouble is:
1. the code doesn't pass the tests (run by Netlify as I understand it)
2. code in ProductPage both in master and feature/general-styles should be the same as I understand - but is it true only for origin branches? It wasn't the same locally, so I pull for my local feature/general-styles  
3. And now I can see where the bug is in feature/general-styles (I should go with a closing fragment `</>` rather that a closing `</div>` ) - but I'm so confused right now, that I don't know what to do, not to make a greater mess! 😛 

I need to go pick Radek up, so I decided to write it all down before I forget what's happened and try to work it out tomorrow.

**Comment by fromITzerotoIThero** [comment]
I would really appreciate it, I have a feeling like I'm doing more harm than good when dealing with new Git/GitHub issues on my own 🤦 😛  I guess I don't have a profound understanding of what's going on and that's why things keep breaking :( 
Monday, as usual, will do?
By then I'll work on my personal website :)

**Comment by piotrwicijowski** [comment]
Let's do it on Monday, see you then!

File: package.json [hunk 1]
```diff
@@ -7,13 +7,15 @@
...
     "@mui/material": "^5.11.4",
+    "@reduxjs/toolkit": "^1.9.5",
     "@supabase/supabase-js": "^2.1.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "classnames": "^2.3.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.1.0",
```
Comment (piotrwicijowski) [inline package.json:18]: Why did you make changes with redux if this PR is just for styles?

Comment (fromITzerotoIThero) [reply package.json:18]: I'm a fool!

**Review by piotrwicijowski — APPROVED** [review]

**Pull request merged**